### PR TITLE
"if(" --> "if (" to conform to LSST standard (DM-3888)

### PR DESCRIPTION
**Pull request by jbecla**

### CODE REVIEW

File: core/modules/parser/FromFactory.cc [hunk 1]
```diff
@@ -309,7 +309,7 @@ class FromFactory::RefGenerator {
     /// ( "inner" | outer_join_type ("outer")? )? "join" table_ref join_spec
     // Takes ownership of @param left
     query::JoinRef::Ptr _makeJoinWithSpec(RefAST sib) const {
-        if(!sib.get()) {
+        if (!sib.get()) {
             throw ParseException("Null JOIN_WITH_SPEC sibling", sib); }
```
Comment (fritzm) [inline core/modules/parser/FromFactory.cc]: well, while you're at it, make either single line or move } to its own line here? :)

File: core/modules/parser/FromFactory.cc [hunk 2]
```diff
@@ -417,31 +417,31 @@ class FromFactory::RefGenerator {
...
         case SqlSQL2TokenTypes::JOIN_CONDITION:
             token = specToken->getFirstChild();
-            if(!token.get()
+            if (!token.get()
                || token->getType() != SqlSQL2TokenTypes::SQL2RW_on) {
                 throw ParseException("Expected ON in join condition", specToken);
             }
             token = token->getNextSibling();
-             if(!token.get()
+             if (!token.get()
```
Comment (fritzm) [inline core/modules/parser/FromFactory.cc]: indentation?

File: core/modules/parser/FromFactory.cc [hunk 3]
```diff
@@ -489,9 +489,9 @@ class FromFactory::RefGenerator {
     ///    column_name (COMMA column_name)*
     /// ;
     std::shared_ptr<query::ColumnRef> _processColumn(RefAST sib) const {
-        if(!sib.get()) {
+        if (!sib.get()) {
             throw ParseException("NULL column node", sib); }
-        if(sib->getType() != SqlSQL2TokenTypes::REGULAR_ID) {
+        if (sib->getType() != SqlSQL2TokenTypes::REGULAR_ID) {
             throw ParseException("Bad column node for USING", sib); }
```
Comment (fritzm) [inline core/modules/parser/FromFactory.cc]: well, I guess these are fairly readable if there are several in a row this way, and you can at least set breakpoints on them this way, so maybe disregard comment above?

File: core/modules/qproc/SecondaryIndex.cc [hunk 4]
```diff
@@ -229,7 +229,7 @@ class FakeBackend : public SecondaryIndex::Backend {
             return (c.name == "sIndex" || c.name == "sIndexBetween"); }
     };
     bool _hasSecondary(query::ConstraintVector const& cv) {
-        return cv.end() != std::find_if(cv.begin(), cv.end(), _checkIndex());
+        return cv.end() != std::find_if (cv.begin(), cv.end(), _checkIndex());
```
Comment (fritzm) [inline core/modules/qproc/SecondaryIndex.cc]: Nope! :)

File: core/modules/query/ValueExpr.cc [hunk 5]
```diff
@@ -271,17 +271,17 @@ std::ostream& operator<<(std::ostream& os, ValueExpr const& ve) {
 }
 
 std::ostream& operator<<(std::ostream& os, ValueExpr const* ve) {
-    if(!ve) return os << "<NULL>";
+    if (!ve) return os << "<NULL>";
     return os << *ve;
 }
```
Comment (fritzm) [inline core/modules/query/ValueExpr.cc]: How about just
```C++
return os << (ve ? *ve : "<NULL>");
```

File: core/modules/util/EventThread.cc [hunk 6]
```diff
@@ -99,7 +99,7 @@ PoolEventThread::Ptr ThreadPool::release(PoolEventThread *thrd) {
     PoolEventThread::Ptr thrdPtr;
     {
         std::lock_guard<std::mutex> lock(_poolMutex);
-        auto iter = std::find_if(_pool.begin(), _pool.end(), func);
+        auto iter = std::find_if (_pool.begin(), _pool.end(), func);
```
Comment (fritzm) [inline core/modules/util/EventThread.cc]: nope!

File: core/modules/wdb/QueryRunner.cc [hunk 7]
```diff
@@ -378,7 +378,7 @@ bool QueryRunner::_dispatchChannel() {
 void QueryRunner::cancel() {
     LOGS(_log, LOG_LVL_WARN, "Trying QueryRunner::cancel() call, experimental");
     _cancelled.store(true);
-    if(!_mysqlConn.get()) {
+    if (!_mysqlConn.get()) {
     LOGS(_log, LOG_LVL_WARN, "QueryRunner::cancel() no MysqlConn");
```
Comment (fritzm) [inline core/modules/wdb/QueryRunner.cc]: indent

File: core/modules/qdisp/Executive.cc [hunk 8]
```diff
@@ -175,9 +175,9 @@ bool Executive::join() {
     int sCount = 0;
     {
         std::lock_guard<std::recursive_mutex> lock(_jobsMutex);
-        sCount = std::count_if(_jobMap.begin(), _jobMap.end(), successF::f);
+        sCount = std::count_if (_jobMap.begin(), _jobMap.end(), successF::f);
```
Comment (fritzm) [inline core/modules/qdisp/Executive.cc]: nope!

File: core/modules/query/ValueExpr.cc (hunk 5 above)
Comment (jbecla) [inline core/modules/query/ValueExpr.cc]: But ValueExpr does not have operator?, so going that route would be more trouble than it is worth it.

Comment (fritzm) [inline core/modules/query/ValueExpr.cc]: But this case is just a nullptr check, isn't it?  It's a very minor concern either way, ha!

**Pull request closed without merging**